### DISCUSSION TITLE
No PPM option found in INAV 6 configurator for operating Flysky FS-i6S with FS-IA10B receiver

### DISCUSSION THREAD

**Discussion by Pratham340** [iNavFlight/inav-configurator#1770] [General]
Connected PPM single wire to Mateksys F722-WING (RX2) worked just fine with Betaflight but when tried with INAV there was no option for PPM and just wasn't working. Any help from anybody would be highly appreciated! :)
![Screenshot 2023-04-28 143630](https://user-images.githubusercontent.com/71821987/235107011-e1a4fec9-9d69-4a0f-b597-6037de88e47e.png)
![Screenshot 2023-04-28 143608](https://user-images.githubusercontent.com/71821987/235107015-d633e3a2-c4ef-41ef-bed4-6decbfb763c5.png)

**Comment by MrD-RC** [comment]
PPM has been removed. I’m sure it was removed in the previous version too. It was in the release notes.
